python/qa-lint/action.yaml

```diff
@@ -1,26 +1,44 @@
 name: Static analysis
+inputs:
+  python-version:
+    required: false
+    description: Version of python
+    default: "3.12.2"
+  poetry-version:
+    required: false
+    description: Version of poetry
+    default: "1.8.2"
```

Comment:
poetry is used in this action? ruff is working standalone you nid only pip to install it

Reply:
of course, it's no used. removed.